jokes/bonk.py

```diff
@@ -0,0 +1,68 @@
+import discord
+import logging
+import re
+from profanity_check import predict
+from redbot.core.bot import Red
+
+from .joke import Joke
+from .util import random_image, BONK_DIR
+
+LOG = logging.getLogger("red.dad")
+
+class BonkJoke(Joke):
+    def __init__(self):
+        """Init for the bonk joke.
+
+        If someone gets a little NSFW, Dad will hastily correct them
+        and send them to horny jail with a swift bonk.
+        
+        library for profanity filter
+        https://pypi.org/project/profanity-check/
+
+        predict([msg]) Returns
+        ----------------------
+        True
+            Message is profain
+        False
+            Message is clean
+        """
+        
+        # Set up super class
+        super().__init__("bonk", 100.0)

```

Comment:
I recommend having the default be significantly less than 100% as people tend to swear in chats quite frequently for benign reasons. I would recommend 20% as max, but maybe lower like 10% or 5%.